Assets/Scripts/Customers/Customer.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Customer : MonoBehaviour
+{
+    [SerializeField] private float _walkSpeed;
+    [SerializeField] private float _despawnCustomerPointOnX = 11f;
```

Comment:
Can we not prefix the SerializeField ones with underscore? Conventional non-serialized private fields can have the underscore though.

Reply:
- serialized private and local var without underscore.
- non-serialized private with underscore.
- public without underscore and stating with Captital letter.
right?

Reply:
Yes right. Public ones being necessarily properties.
Example: 
```c#
public int WalkSpeed { get; private set; }
```
Above example is what you'd do to access the SO field on the `Customer` btw.

Reply:
Not sure what you mean, we create on SO this line `public int WalkSpeed { get; private set; }` in order to be able to access  WalkSpeed on Customers script?

Reply:
You'd have a serialized field to set in the SO inspector:
```c#
[SerializeField] private float walkSpeed;
```
and a public property for other classes to read it:
```c#
public float WalkSpeed => walkSpeed;
```